GitCommands/Git/GitModule.cs

```diff
@@ -3141,12 +3141,12 @@ public IReadOnlyList<string> GetAllBranchesWhichContainGivenCommit(ObjectId obje
                 return Array.Empty<string>();
             }
 
-            var result = exec.StandardOutput.Split(new[] { '\r', '\n', '*', '+' }, StringSplitOptions.RemoveEmptyEntries);
+            string[] result = exec.StandardOutput.Split(new[] { '\r', '\n' }, StringSplitOptions.RemoveEmptyEntries);
 
             // Remove symlink targets as in "origin/HEAD -> origin/master"
-            for (var i = 0; i < result.Length; i++)
+            for (int i = 0; i < result.Length; i++)
             {
-                var item = result[i].Trim();
+                string item = Regex.Replace(result[i], @"^[*+ ] ", "");
```

Comment:
This should maybe be trimmed with code instead, or we should increase the static regex cache (15 entries)
I do not request changes though

As discussed above, a comment about the format should be added.

Reply:
All good changes